conf/app.routes

```diff
@@ -37,7 +37,7 @@ POST          /form/addtolist/remove/:formTemplateId/:maybeAccessCode/:sectionNu
 GET           /summary/pdf/:formTemplateId                                                                       uk.gov.hmrc.gform.gform.SummaryController.downloadPDF(formTemplateId: FormTemplateId, a: Option[AccessCode])
 GET           /summary/:formTemplateId/                                                                          uk.gov.hmrc.gform.gform.SummaryController.summaryById(formTemplateId: FormTemplateId, a: Option[AccessCode])
 GET           /summary/:formTemplateId/:maybeAccessCode                                                          uk.gov.hmrc.gform.gform.SummaryController.summaryById(formTemplateId: FormTemplateId, maybeAccessCode: Option[AccessCode])
-POST          /summary/:formTemplateId/:maybeAccessCode                                                          uk.gov.hmrc.gform.gform.SummaryController.submit(formTemplateId: FormTemplateId, maybeAccessCode: Option[AccessCode], action: Direction)
+POST          /summary/:formTemplateId/:maybeAccessCode                                                          uk.gov.hmrc.gform.gform.SummaryController.submit(formTemplateId: FormTemplateId, maybeAccessCode: Option[AccessCode], action: Direction, refreshToken: Option[String])
```

Comment:
I don't see any reason why `refreshToken` should be an `Option`. Maybe I'm missing something?